heartbeat/redis

```diff
@@ -324,6 +324,7 @@ function start() {
 
 	[[ ! -d "$REDIS_RUNDIR" ]] && mkdir -p "$REDIS_RUNDIR"
 	chown -R "$REDIS_USER" "$REDIS_RUNDIR"
+	restorecon -Rv "$REDIS_RUNDIR"
```

Comment:
What if SELinux is not available? In that `restorecon` won't be there and this will crash, right?
Shouldn't this be guarded by `have_binary`?